egs/iwslt22_ta/ST/zipformer/beam_search.py

```
@@ -1203,17 +1203,17 @@ def modified_beam_search_hat(
 
         logits = logits.squeeze(1).squeeze(1)  # (num_hyps, vocab_size)
 
-       
+
         # For blank symbol, log-prob is log-sigmoid of the score
         logp_b = torch.nn.functional.logsigmoid(logits[..., 0])
         # Additionally, to ensure the the probs of blank and non-blank sum to 1, we
         # need to add the following term to the log-probs of non-blank symbols. This
         # is equivalent to log(1 - sigmoid(logits[..., 0])).
         breakpoint()
```

Ruff (0.14.13)

Trace found: `breakpoint` used

(T100)